OpenSim/Actuators/Thelen2003Muscle.cpp

```diff
@@ -1623,9 +1670,9 @@ SimTK::Vector Thelen2003Muscle::
     }
 
     if(abs(ferr) < tolerance){
-        result[0] = 1.0;
-        result[1] = aFalFv/(aAct*aFal);
+        result = max(0.0, aFalFv/(aAct*aFal));
+        return result;
     }
 
-    return result;
+    OPENSIM_THROW_FRMOBJ(Exception, "Failed to calculate fiber-velocity.");
```

Comment:
I made the following comment in the first version of this PR:
"Can any kind of suggestion be provided here? At least something like "Fiber-velocity solver failed to converge" would be more descriptive ("Failed to calculate…" could mean a lot of things)."
My comment was not addressed and no explanation was provided. I think it is still a valid comment, but obviously is not critical.